Apps/Inc/IgnitionContactor.h

```diff
@@ -0,0 +1,3 @@
+#include "os.h"
+#include "common.h"
+#include "Tasks.h"
```

Comment:
Don't need an application-specific header for this. Just move these into the app itself.